frontend/src/core/loaders/components/SkeletonLoader.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import './SkeletonLoader.css';
+
+const list = [...Array(30).keys()];
+
+const SkeletonLoader = () => (
```

Comment:
To stay consistent with the rest of the app, please use the `function` syntax here. That way you can write…

```js
export default function SkeletonLoader() {
```

… and avoid the export on a different line.